src/component/ContentData.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import VideoList from './VideoList';
+import { Grid } from 'semantic-ui-react';
+import youtube from '../api/youtube';
+import onFormSubmit from './FormSubmit';
+import '../style/Grid.css';
+import '../style/VideoGrid.css';
+
+class ContentData extends React.Component {
+
+   state = { videos: [] };
+
+
+   componentDidMount() {
+
+      onFormSubmit(this.props.category, 4).then(response => {
```

Comment:
@RajanTank What is this `4`? Remember - Always take constants for static numbers or string which are meaningful so that you don't miss what is it after few months.